test/test_calculus.py

```diff
@@ -24,5 +24,10 @@ def test_calculus():
             "{{a -> 1}}",
             "Issue #1168",
         ),
+        (
+            "v1 := Exp[x] - 3x; v2 = {x, 2}; v2 = {x, 2}; FindRoot[v1, v2]",
```

Comment:
why v2 is defined two times?

Reply:
This was a cut and paste typo. Should be fixed in  98221899